src/main/java/fr/esgi/al5_2/Tayarim/controllers/AuthController.java

```diff
@@ -19,84 +19,87 @@
 @RequestMapping("/auth")
 public class AuthController {
 
-    private final AuthService authService;
+  private final AuthService authService;
 
-    public AuthController(AuthService authProprietaire) {
-        this.authService = authProprietaire;
-    }
-
-    @PostMapping("/login")
-    public ResponseEntity<AuthLoginResponseDTO> loginProprietaire(@Valid @RequestBody AuthLoginDTO authLoginDTO){
-        return new ResponseEntity<>(
-                authService.loginProprietaire(authLoginDTO.getEmail(), authLoginDTO.getMotDePasse()),
-                HttpStatus.OK
-        );
-    }
-
-    @GetMapping("")
-    public ResponseEntity<AuthLoginResponseDTO> authProprietaire(@RequestHeader("Authorization") String authHeader){
+  public AuthController(AuthService authProprietaire) {
+    this.authService = authProprietaire;
+  }
 
-        String jwtToken = getTokenFromHeader(authHeader);
+  @PostMapping("/login")
+  public ResponseEntity<AuthLoginResponseDTO> login(@Valid @RequestBody AuthLoginDTO authLoginDTO) {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'authLoginDTO' must contain no more than '1' consecutive capital letters.